cmd/agent/common/import.go

```diff
@@ -92,18 +102,25 @@ func ImportConfig(oldConfigDir string, newConfigDir string, force bool) error {
 	}
 
 	for _, f := range files {
-		if f.IsDir() || filepath.Ext(f.Name()) != ".yaml" {
+		if f.IsDir() || filepath.Ext(f.Name()) != cfgExt {
 			continue
 		}
+		checkName := f.Name()[:len(f.Name())-len(cfgExt)]
```

Comment:
`strings.TrimRight` would be easier to read and returns a slice without allocating a new string. Same for other occurences